src/python/pants/pantsd/pants_daemon_core.py

```diff
@@ -152,7 +152,6 @@ def prepare(
         options_fingerprint = OptionsFingerprinter.combined_options_fingerprint_for_scope(
             GLOBAL_SCOPE,
             options_bootstrapper.bootstrap_options,
-            invert=True,
```

Comment:
It looks like the reason that this was `invert=True` before was that it was fingerprinting everything, and setting the "default" of `fingerprint` to True (whereas it would otherwise have been False)

It looks like you've set the default of `fingerprint` to True in general, so yea: makes sense.

Reply:
Exactly. It was very non-obvious what was going on before.